src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -940,6 +941,8 @@ private synchronized void regenerateAITasks() {
 	@Override protected AggregatedValidationResult validatePage(int page) {
 		if (page == 0) {
 			return new AggregatedValidationResult(mobModelTexture, mobName);
+		} else if (page == 1) {

```

Comment:
If you insert new page, indices of other pages change. This feels like this part of PR was not tested